src/GraphQL/Types/EnumerationGraphType.cs

```diff
@@ -33,7 +33,8 @@ public void AddValue(EnumValue value)
         public override object Coerce(object value)
         {
             var found = Values.FirstOrDefault(v => v.Value.Equals(value));
-            return found != null ? found.Name : null;
+            if (found != null) return found.Name;
+            return Values.FirstOrDefault(v => v.Name.Equals(value))?.Value;
```

Comment:
I don't want to accept this change without a use case and unit tests.  Is this how the reference javascript project works?  I'm not sure if it is or not.  I do know we potentially have some issues here, though I don't want to make changes without locking down the behavior.